src/com/company/Main.java

```diff
@@ -50,7 +50,7 @@ public static void checkforsupport(File file)
                 }
         }
 
-        public static void checkforexistance(File file) {
+        public static void checkForExistance(File file) {
```

Comment:
You should ideally be throwing an exception from here or atleast return false if the file is not in a supported format

Reply:
i have made to print as "file is not valid" if it is not in a supported format.